kits/csharp/simple/main.py

```diff
@@ -0,0 +1,68 @@
+from subprocess import Popen, PIPE
+from threading  import Thread
+from queue import Queue, Empty
+
+import atexit
+import os
+import sys
+agent_processes = [None, None]
+t = None
+q = None
+def cleanup_process():
+    global agent_processes
+    for proc in agent_processes:
+        if proc is not None:
+            proc.kill()
+def enqueue_output(out, queue):
+    for line in iter(out.readline, b''):
+        queue.put(line)
+    out.close()
+def cpp_agent(observation, configuration):
+    """
+    a wrapper around a java agent
+    """
+    global agent_processes, t, q
+
+    agent_process = agent_processes[observation.player]
+    ### Do not edit ###
+    if agent_process is None:
+        if "__raw_path__" in configuration:
+            cwd = os.path.dirname(configuration["__raw_path__"])
+        else:
+            cwd = os.path.dirname(__file__)
+        agent_process = Popen(["dotnet", "./bin/Release/*/Bot.dll"], stdin=PIPE, stdout=PIPE, stderr=PIPE, cwd=cwd)
```

Comment:
This is dependent on having the dotnet runtime installed on the running instance. You may need to use the targeted executable (**./bin/Release/netcoreapp3.1/linux-x64/Bot**)